src/app/components/MediaLoader/Placeholder/PlayButton/index.tsx

```diff
@@ -39,7 +39,7 @@ const PlayButton = ({
       type="button"
       css={styles.button}
       onClick={onClick}
-      {...(className && { className })}
+      {...(className ? { className } : undefined)}
```

Comment:
So, basically this (and similar changes) are making sure we are following the correct spread pattern for React 19 because in 19 dev mode crashes when false is evaluated during spread?